x-pack/plugins/observability_solution/logs_explorer/README.md

```diff
@@ -1,10 +1,10 @@
-# Log Explorer
+# Logs Explorer
 
-This plugin is home to the `<LogExplorer />` component and related types. It implements several of the underlying concepts that the [Observability Log Explorer app](../observability_solution/observability_logs_explorer) builds upon.
+This plugin is home to the `<LogExplorer />` component and related types. It implements several of the underlying concepts that the [Observability Logs Explorer app](../observability_solution/observability_logs_explorer) builds upon.
```

Comment:
**nit**: shall we rename also the component here and in the implementation to `LogsExplorer`?

Reply:
Definitely, there are quite a few things that still have the singular name, I have started renaming these in batches to avoid major conflicts and long-running PRs. This PR is mostly for tests and the app url.